packages/amplication-server/src/core/workspace/workspace.controller.ts

```diff
@@ -59,4 +62,24 @@ export class WorkspaceController {
       quantity
     );
   }
+
+  @Post(`createWorkspacesResourcesDefaultCustomDtosMigration/:token`)
+  async createWorkspacesResourcesDefaultCustomDtosMigration(
+    @Param("token") token: string,
+    @Body()
+    data: CreateWorkspacesResourcesDefaultCustomDtosMigrationInput
+  ): Promise<boolean> {
+    this.logger.info("createWorkspacesResourcesDefaultCustomDtosMigration....");
+    const { quantity } = data;
+
+    if (
+      this.configService.get<string>("CUSTOM_DTO_MIGRATION_TOKEN") !== token
```

Comment:
We need a "twin" PR in cluster-configuration that adds CUSTOM_DTO_MIGRATION_TOKEN to all envs, based on an AWS secret. We can do that part together.

Reply:
Changed to the existing var for custom-action migrations